src/components/localisation/routes.ts

```diff
@@ -0,0 +1,18 @@
+// import languages from '../../../localisation/languages.json'
+import path from 'path'
+import { getAppEntryPointDir } from '../../components/utilityFunctions'
+import config from '../../config'
+
+const { localisationsFolder } = config
+
+export const routeLocalisations = async (request: any, reply: any) => {
+  reply.sendFile('languages.json', path.join(getAppEntryPointDir(), localisationsFolder))
+}
+
+export const routeGetLanguageFile = async (request: any, reply: any) => {
+  const { code } = request.params
+  const stringsFile = path.join(code, 'strings.json')
+  reply.sendFile(stringsFile, path.join(getAppEntryPointDir(), localisationsFolder))
+}
+
+export const updateLanguageFiles = async (request: any, reply: any) => {}
```

Comment:
Hmmm is there a plan for this one? Not sure why e would use an update here?

Reply:
Because at some point we need a mechanism to write language files back to the server, so this was just meant as a placeholder until we have a spec for how that will work. I figured we'd build the API for it before the UI.